src/pages/login/LoginPage.scss

```diff
@@ -1,21 +1,38 @@
-// @import '../../styles/styles';
 @import '../../styles/variables';
 
+#normal_login {
+    margin-top: 200px;
+}
+
+.Logo {
```

Comment:
Don't use `camelCase` for css class names, `Logo` and `logo` is a bad name, replace `Logo` with `logo-wrapper`